backend/clubs/views.py

```diff
@@ -2878,33 +2878,64 @@ def get(self, request):
                                     type: integer
         ---
         """
-        id = request.query_params.get("event")
-        if not id:
-            return Response({"count": 0})
-
-        try:
-            conn = get_redis_connection()
-        except NotImplementedError:
-            return Response({"count": 0})
-        key = f"zoom:meeting:live:{id}"
-        ans = conn.get(key)
-        if ans is None:
-            ans = 0
-        else:
-            ans = int(ans)
-        return Response({"count": ans})
+        meeting_id = request.query_params.get("event")
+        return Response({"count": ZoomMeetingVisit.objects.filter(meeting_id=meeting_id).count()})
 
     def post(self, request):
         action = request.data.get("event")
-        meeting_id = request.data.get("payload", {}).get("object", {}).get("id", None)
-        if meeting_id is not None:
-            conn = get_redis_connection()
-            key = f"zoom:meeting:live:{meeting_id}"
-            if action == "meeting.participant_joined":
-                conn.incr(key)
-            elif action == "meeting.participant_left":
-                conn.decr(key)
-            conn.expire(key, datetime.timedelta(hours=8))
+        if action == "meeting.participant_joined":
+            email = (
+                request.data.get("payload", {})
+                .get("object", {})
+                .get("participant", {})
+                .get("email", None)
+            )
+            person = get_user_model().objects.filter(email=email)
+
+            event_name = request.data.get("payload", {}).get("object", {}).get("topic", None)
+            event = Event.objects.filter(name=event_name).order_by("-start_time")[0]
+
+            meeting_id = request.data.get("payload", {}).get("object", {}).get("id", None)
+            participant_id = (
+                request.data.get("payload", {})
+                .get("object", {})
+                .get("participant", {})
+                .get("user_id", None)
+            )
+            join_time = (
+                request.data.get("payload", {})
+                .get("object", {})
+                .get("participant", {})
+                .get("join_time", None)
+            )
+
+            if event:
+                club = event.club
+                ZoomMeetingVisit.objects.create(
+                    person=person,
+                    club=club,
+                    event=event,
```

Comment:
You originally defined event as a ManyToMany field, but you use it like a foreign key here, which would throw an error.

I would recommend writing a test case to ensure that the behavior is what you want. You have the sample input format in the clubhouse post (replace any sensitive user information with dummy values).